### PR TITLE
chore: fix security/issues/112

### PR DESCRIPTION
**Pull request by lucasgonze**
Address tasks in https://github.com/magma/security/issues/112

Signed-off-by: Lucas Gonze <lucas@gonze.com>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by emakeev — APPROVED** [review]
LGTM

**Review by emakeev — APPROVED** [review]

File: feg/radius/src/config/samples/radius.tataipx.config.json
Comment (crasu) [inline feg/radius/src/config/samples/radius.tataipx.config.json:28]: This will still be contained in git history - is someone taking care of chaging the credentials of this server/taking it offline?

**Review by crasu — APPROVED** [review]

File: feg/radius/src/config/samples/radius.tataipx.config.json
Comment (lucasgonze) [reply feg/radius/src/config/samples/radius.tataipx.config.json:28]: Yes, I'm working on it.

Note that this server doesn't belong to us.

**Comment by lucasgonze** [comment]
Since this PR does have an approval from a member of approvers-cwf, and it is blocked on a non-existent group, I will mark it ready2merge and skip the formalities.

**Comment by ssanadhya** [comment]
@aharonnovo , could you please stamp this PR? I understand that this code will not be maintained going forward, so created a clean up PR https://github.com/magma/magma/pull/13545 , cc: @lucasgonze

**Comment by ssanadhya** [comment]
@lucasgonze , I just merged #13545 . Could you please rebase this on the latest master? It should be good to merge then without needing approval for `xwf`.

**Comment by lucasgonze** [comment]
I have rebased and repushed. There is one failing test but it is unrelated to these changes.

**Pull request merged**
